docs/SDD/02.overview/00.overview.adoc

```diff
@@ -177,6 +178,6 @@ On the other hand, this baseline functionality is desirable to allow PEP-chainin
 
 === Request Forwarding with JWT header
  
-After validating the RPT we proceed to make a call to the introspection endpoint (/oxauth/restv1/rpt/status) passing through parameters the RPT and the pat. Returning a JSON with the information for that token, called claims, where the user name can appear, for example.
-
+After validating the RPT we proceed to verify the signature of the JWT if the signature verification is enabled in the environment variables and the we pass the JWT to the request header in the request to the resource server.
```

Comment:
Same comment as above, perhaps add description of what happens if the verification is disabled?